app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.droidjump;
+
+import android.content.Context;
+import android.view.SurfaceView;
+
+public class GameView extends SurfaceView implements Runnable {
+
+    public GameView(Context context) {
+        super(context);
+    }
+
+    @Override
+    public void run() {
+
+    }
```

Comment:
nit: if method is empty for now, make it a one-liner:

```
public void run() {}
```

Reply:
Solved